packages/date-picker/src/FilterDateRangePicker/components/Trigger/FilterTriggerButton/FilterTriggerButton.tsx

```diff
@@ -1,33 +1,53 @@
 import React, { forwardRef } from "react"
-import { TriggerButtonBase, TriggerButtonBaseProps } from "../TriggerButtonBase"
+import classnames from "classnames"
+import { Icon } from "@kaizen/component-library"
+import chevronDown from "@kaizen/component-library/icons/chevron-down.icon.svg"
+import chevronUp from "@kaizen/component-library/icons/chevron-up.icon.svg"
+import { FilterBaseButton, FilterBaseButtonProps } from "../FilterBaseButton"
 import styles from "./FilterTriggerButton.module.scss"
 
 export interface FilterTriggerButtonProps
-  extends Omit<TriggerButtonBaseProps, "children"> {
+  extends Omit<FilterBaseButtonProps, "children"> {
   label: string
   selectedValue?: string
+  isOpen?: boolean
 }
 
 export const FilterTriggerButton = forwardRef<
   HTMLButtonElement,
   FilterTriggerButtonProps
->(({ label, selectedValue, ...restProps }, ref) => {
-  const hasSelectedValue = selectedValue && selectedValue !== ""
+>(
+  (
+    { label, selectedValue, isOpen = false, classNameOverride, ...restProps },
+    ref
+  ) => {
+    const hasSelectedValue = selectedValue && selectedValue !== ""
```

Comment:
Could you apply the empty string type guard to prevent this from being an issue?

Reply:
This is essentially a type guard lol. Is there a util out there??

Reply:
I think Nat wrote one, but probably not worth digging it up for this reason lol